cogs5e/sheets/beyond.py

```diff
@@ -280,8 +280,11 @@ def get_skills_and_saves(self):
                     skills[skill_name] = Skill(charval['value'])
                     ignored_ids.add(charval['valueId'])  # this must be the final value so we stop looking
                 elif charval['typeId'] in {24, 25}:  # PROBABLY skill magic/misc bonus
-                    skills[skill_name].value += charval['value']
-                    skills[skill_name].bonus += charval['value']
+                    try:
```

Comment:
similarly, it seems more elegant to LBYL here in the conditional - that would also catch the type 23 case